### DISCUSSION TITLE
Supported editors?

### DISCUSSION THREAD

**Discussion by dgudim** [deckerst/aves#1213] [Q&A]
What photo editing apps are supported? When I am clicking the edit button, it says that "no app supports this action" 🤔

**Comment by deckerst** [comment]
It's not a white list of known apps. Instead, Aves shares the media item with a specific ["edit" intent](https://developer.android.com/reference/android/content/Intent#ACTION_EDIT), and specifying whether the item is an `image` or a `video`.

Most well behaved editor apps will know how to handle that intent and automatically be listed among the available editor apps when the user presses "edit" (for example, Google Photos or Snapseed). On the contrary, some editor apps may be unavailable on purpose, like the image editor from Samsung which is only available when launched from the Samsung Gallery.

**Comment by dgudim** [comment]
That was weird, I just rebooted my phone and everything works, 🙃. Sorry for bothering you
